README.md

```diff
@@ -276,14 +276,24 @@ It supports the Archethic Cryptography rules which are:
   <summary>Remote Endpoint calls</summary>
   <br/>
 
-  #### getOriginKey(endpoint, authorizedPublicKey, privateKey)
-  Query a node to get the origin private key encrypted by the `authorizedPublicKey`. This origin private key is used to sign the transaction (see originSign).
+  #### getOriginKey()
+  Return the hardcoded origin private key for software, this is used fot signing transaction (see OriginSign).
```

Comment:
Just some typo fix: `for signing`.

Also we might put this function in the crypto section maybe.